.github/workflows/snyk_monitor.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Snyk Monitor
+on:
+  push:
+    branches:
+      - main
+jobs:
+  snyk_monitor:
+    if: ${{ github.repository_owner == 'puppetlabs' }}
+    runs-on: ubuntu-latest
+    name: Snyk Monitor
+    steps:
+      - name: Checkout current PR
+        uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Install dependencies
+        run: bundle install --jobs 3 --retry 3
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/ruby@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_PE_TOKEN }}
```

Comment:
I see this token is already in the repo, but since this repo is public shouldn't it be using `SNYK_FOSS_KEY`?

Reply:
Yeah the separation between foss and pe doesn't make sense to me... I was following the `puppet-runtime` example https://github.com/puppetlabs/puppet-runtime/blob/master/.github/workflows/snyk_scan.yaml#L21-L22

Reply:
I think we should go with "public gems" get reported under `FOSS` and everything else is `PE` including pxp-agent-vanagon, puppet-runtime, etc